test/models/gemini_connection_test.rb

```diff
@@ -20,6 +20,17 @@ class GeminiConnectionTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'after destroy hook' do
+    let(:gemini_connection) { gemini_connections(:connection_with_token) }
+    let(:publisher) { publishers(:gemini_completed) }
+    test 'publisher no longer has a selected wallet provider' do
+      assert publisher.selected_wallet_provider, gemini_connection
```

Comment:
Did you mean `assert_equal` or just wanted to `assert` multiple items? If the latter I think you'll need separate statements.

Reply:
`assert_equal`. Thanks, I updated the test.